### PR TITLE
DIV-6424: DN refusal release

### PR DESCRIPTION
**Pull request by yarpo**
DN refusal release.
https://tools.hmcts.net/jira/browse/DIV-6424

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-prod.json [hunk 1]
```diff
@@ -1,9 +1 @@
-[
-  {
-    "LiveFrom": "31/08/2018",
-    "CaseTypeID": "DIVORCE",
-    "CaseFieldID": "PreviousIssueDate",
-    "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "R"
-  }
-]
+[]
```
Comment (yarpo) [inline definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-prod.json:1]: I will remove it

File: definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-solicitorDn-nonprod.json [hunk 2]
```diff
@@ -1,65 +1 @@
...
-  },
-  {
-    "LiveFrom": "14/07/2020",
-    "CaseTypeID": "DIVORCE",
-    "CaseFieldID": "AmendedCaseId",
-    "UserRole": "caseworker-divorce-courtadmin",
-    "CRUD": "R"
-  }
-]
+[]
```
Comment (yarpo) [inline definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-solicitorDn-nonprod.json:1]: I will remove this file.

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields-solicitorDn-nonprod.json [hunk 3]
```diff
@@ -1,74 +1 @@
...
-    "CaseFieldID": "LabelAmendPetitionForRefusalSolPara-6",
-    "PageFieldDisplayOrder": 6,
-    "DisplayContext": "READONLY",
-    "PageID": "amendpetitionstart",
-    "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "No"
-  }
-]
+[]
```
Comment (yarpo) [inline definitions/divorce/json/CaseEventToFields/CaseEventToFields-solicitorDn-nonprod.json:1]: I will remove this file.

File: definitions/divorce/json/CaseField/CaseField-prod.json [hunk 4]
```diff
@@ -1,10 +1 @@
...
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "DIVORCE",
-    "ID": "LabelSolPaymentPara-1",
-    "Label": "Amount to pay: **£550**",
-    "FieldType": "Label",
-    "SecurityClassification": "Public"
-  }
-]
\ No newline at end of file
+[]
```
Comment (yarpo) [inline definitions/divorce/json/CaseField/CaseField-prod.json:1]: I will leave this file

File: definitions/divorce/json/CaseField/CaseField-solicitorDn-nonprod.json [hunk 5]
```diff
@@ -1,18 +1 @@
...
-  {
-    "LiveFrom": "14/07/2020",
-    "CaseTypeID": "DIVORCE",
-    "ID": "AmendedCaseId",
-    "Label": "Case ID for Amended case",
-    "FieldType": "CaseLink",
-    "SecurityClassification": "Public"
-  }
-]
+[]
```
Comment (yarpo) [inline definitions/divorce/json/CaseField/CaseField-solicitorDn-nonprod.json:1]: I will remove this file.

File: definitions/divorce/json/CaseTypeTab/CaseTypeTab-prod.json [hunk 6]
```diff
@@ -1,13 +1 @@
...
-    "Channel": "CaseWorker",
-    "TabID": "LinkedCase",
-    "TabLabel": "Linked Cases",
-    "TabDisplayOrder": 24,
-    "CaseFieldID": "PreviousCaseId",
-    "TabFieldDisplayOrder": 1,
-    "TabShowCondition": "PreviousCaseId.CaseReference=\"*\""
-  }
-]
+[]
```
Comment (yarpo) [inline definitions/divorce/json/CaseTypeTab/CaseTypeTab-prod.json:1]: I will leave this file

File: definitions/divorce/json/CaseTypeTab/CaseTypeTab-solicitorDn-nonprod.json [hunk 7]
```diff
@@ -1,24 +1 @@
...
-    "Channel": "CaseWorker",
-    "TabID": "LinkedCase",
-    "TabLabel": "Linked Cases",
-    "TabDisplayOrder": 24,
-    "CaseFieldID": "AmendedCaseId",
-    "TabFieldDisplayOrder": 6,
-    "FieldShowCondition": "AmendedCaseId.CaseReference=\"*\""
-  }
-]
+[]
```
Comment (yarpo) [inline definitions/divorce/json/CaseTypeTab/CaseTypeTab-solicitorDn-nonprod.json:1]: I will remove this file.

File: definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-prod.json (hunk 1 above)
Comment (Dizrupto) [reply definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-prod.json:1]: yep

Comment (yarpo) [reply definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-prod.json:1]: removed.

File: definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json [hunk 8]
```diff
@@ -6,41 +6,6 @@
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
   },
```
Comment (Dizrupto) [inline definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json:8]: Oow one more is missing actually:
AuthorisationCaseState-nonprod.json

**Pull request merged**